x/lockup/client/cli/tx.go

```diff
@@ -72,3 +73,20 @@ func NewSetRewardReceiverAddress() (*osmocli.TxCliDesc, *types.MsgSetRewardRecei
 		Long:  "sets reward receiver address for the designated lock id",
 	}, &types.MsgSetRewardReceiverAddress{}
 }
+
+// NewRebondTokensCmd allows a user to rebond their tokens if they have unlocked them.
+func NewRebondTokensCmd() (*osmocli.TxCliDesc, *types.MsgRebondTokens) {
+	return &osmocli.TxCliDesc{
+		Use:   "rebond-tokens",
```

Comment:
there is no need for this, since arguments are being added dynamically via reflection when building CLI

Reply:
The need is there because if i type "osmosisd tx lockup rebond-tokens -h" I need to see what the inputs are right? Or are you saying the prompt is also automatically edited?

Reply:
yeah exactly! it is automatically edited. I decided to add this functionality since we were getting lots of issues when some CLI command had wrong parameters specified due to human factor.

Reply:
if you are interested how exactly this works, you may find the answers here: https://github.com/osmosis-labs/osmosis/pull/6005

Reply:
Thanks, learned something new :)